### ISSUE TITLE
Loss functions

### ISSUE THREAD

**Issue by ankitkumar8** [liuziwei7/fashion-landmarks#6]
Where are the lossfunctions  of the net? I need help in setting up that weighted L2 loss for landmark localisation

**Comment by yysijie** [comment]
Sorry, we have not released the code for training. Thus you can't find any loss function in prototxt files. In training process, we used “EuclideanLoss” layer in caffe as loss function layer. The loss weight of visibility is 1. And the loss weight of landmark regression is 1 in the first 2000 iterations and 20 in the next 8000 iterations.  The batch size is 40. I hope this is helpful.

**Comment by ankitkumar8** [comment]
Thank You~ :)
Also, how do you go about weighting it with visibility. 
The dynamic parameter for weight i.e. visibility is where I am having a problem in understanding.
Also, in your paper you don't even discuss about landmark visibility having 3 labels, and here suddenly we have a separate label for truncated or cutoff from the photo. :( :( In the paper it seems like only having 2 labels.

**Comment by NeverBehave** [comment]
Is it possible to release the training code?

**Comment by hongge831** [comment]
Is it possible to release the training code?

**Comment by HarryZJ** [comment]
Is it possible to release the training code?
